lib/embulk/input/presto/explain_parser.rb

```diff
@@ -6,7 +6,7 @@ def self.parse(explain_result)
           explain_text = explain_result.flatten.last.lines.first
           column_name_raw, column_type_raw = explain_text.split(' => ')
           names = column_name_raw.split('[').last.split(']').first.split(',').map{ |name| name.strip }
-          types = column_type_raw.split('[').last.split(']').first.split(',').map{ |info| info.split(':').last }
+          types = column_type_raw.split('[').last.split(']').first.gsub(/\(.+?, .+?\)/, "").split(',').map{ |info| info.split(':').last }
```

Comment:
`/\(.*/` will remove some of the needed schema unintentionally.

For example:

```
"data:map(varchar, varchar), user_id:bigint".gsub(/\(.*/, '')
=> "data:map"
```

Above example should be `"data:map, user_id:bigint"`.

But if the schema contains an `array` type, my PR  is not correct. 
So I'll use `/\(.*?\)` instead.

```
irb(main):003:0> "data:map(varchar, varchar), user_id:bigint, data2:array(double)".gsub(/\(.*?,.*?\)/, '')
=> "data:map, user_id:bigint, data2:array(double)"
irb(main):004:0> "data:map(varchar, varchar), user_id:bigint, data2:array(double)".gsub(/\(.*?\)/, '')
=> "data:map, user_id:bigint, data2:array"
```